<!DOCTYPE html>
<html lang="zh-cn">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
    <title>设备管理模块测试报告</title>
    <meta name="generator" content="HtmlTestRunner 0.8.4"/>
    <link rel="stylesheet" href="http://cdn.bootcss.com/bootstrap/3.3.0/css/bootstrap.min.css">
    
<style type="text/css" media="screen">

/* -- css div popup ------------------------------------------------------------------------ */
.popup_window {
    display: none;
    position: relative;
    left: 0px;
    top: 0px;
    /*border: solid #627173 1px; */
    padding: 10px;
    background-color: #99CCFF;
    font-family: "Lucida Console", "Courier New", Courier, monospace;
    text-align: left;
    font-size: 10pt;
    width: 500px;
}

/* -- report ------------------------------------------------------------------------ */

#show_detail_line .label {
    font-size: 85%;
    cursor: pointer;
}

#show_detail_line {
    margin: 2em auto 1em auto;
}

#total_row  { font-weight: bold; }
.hiddenRow  { display: none; }
.testcase   { margin-left: 2em; }

</style>


    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
    <!--[if lt IE 9]>
      <script src="http://cdn.bootcss.com/html5shiv/3.7.2/html5shiv.min.js"></script>
      <script src="http://cdn.bootcss.com/respond.js/1.4.2/respond.min.js"></script>
    <![endif]-->
  </head>
<body>
<script language="javascript" type="text/javascript"><!--
output_list = Array();

/* level - 0:Summary; 1:Failed; 2:All */
function showCase(level) {
    trs = document.getElementsByTagName("tr");
    for (var i = 0; i < trs.length; i++) {
        tr = trs[i];
        id = tr.id;
        if (id.substr(0,2) == 'ft') {
            if (level < 1) {
                tr.className = 'hiddenRow';
            }
            else {
                tr.className = '';
            }
        }
        if (id.substr(0,2) == 'pt') {
            if (level > 1) {
                tr.className = '';
            }
            else {
                tr.className = 'hiddenRow';
            }
        }
    }
}


function showClassDetail(cid, count) {
    var id_list = Array(count);
    var toHide = 1;
    for (var i = 0; i < count; i++) {
        tid0 = 't' + cid.substr(1) + '.' + (i+1);
        tid = 'f' + tid0;
        tr = document.getElementById(tid);
        if (!tr) {
            tid = 'p' + tid0;
            tr = document.getElementById(tid);
        }
        id_list[i] = tid;
        if (tr.className) {
            toHide = 0;
        }
    }
    for (var i = 0; i < count; i++) {
        tid = id_list[i];
        if (toHide) {
            document.getElementById('div_'+tid).style.display = 'none'
            document.getElementById(tid).className = 'hiddenRow';
        }
        else {
            document.getElementById(tid).className = '';
        }
    }
}


function showTestDetail(div_id){
    var details_div = document.getElementById(div_id)
    var displayState = details_div.style.display
    // alert(displayState)
    if (displayState != 'block' ) {
        displayState = 'block'
        details_div.style.display = 'block'
    }
    else {
        details_div.style.display = 'none'
    }
}


function html_escape(s) {
    s = s.replace(/&/g,'&amp;');
    s = s.replace(/</g,'&lt;');
    s = s.replace(/>/g,'&gt;');
    return s;
}

/* obsoleted by detail in <div>
function showOutput(id, name) {
    var w = window.open("", //url
                    name,
                    "resizable,scrollbars,status,width=800,height=450");
    d = w.document;
    d.write("<pre>");
    d.write(html_escape(output_list[id]));
    d.write("\n");
    d.write("<a href='javascript:window.close()'>close</a>\n");
    d.write("</pre>\n");
    d.close();
}
*/
--></script>

<div class="container">
    <div class='heading'>
<h1>设备管理模块测试报告</h1>
<p><strong>Start Time:</strong> 2017-03-30 14:13:52</p>
<p><strong>Duration:</strong> 0:01:57.910199</p>
<p><strong>Status:</strong> <span class="text text-warning">Error <strong>23</strong></span></p>

<p class='description'>用例执行情况：</p>
</div>


    
<p id='show_detail_line'>
<span class="label label-primary" onclick="showCase(0)">Summary</span>
<span class="label label-danger" onclick="showCase(1)">Failed</span>
<span class="label label-default" onclick="showCase(2)">All</span>
</p>
<table id='result_table' class="table">
    <thead>
        <tr id='header_row'>
            <th>Test Group/Test case</td>
            <th>Count</td>
            <th>Pass</td>
            <th>Fail</td>
            <th>Error</td>
            <th>View</td>
        </tr>
    </thead>
    <tbody>
        
<tr class='text text-warning'>
    <td>test_case_085_dev_manage_dev_stock.TestCase086DevManageDevSearchByIMEI</td>
    <td>1</td>
    <td>0</td>
    <td>0</td>
    <td>1</td>
    <td><a class="btn btn-xs btn-primary"href="javascript:showClassDetail('c1',1)">Detail</a></td>
</tr>

<tr id='ft1.1' class='text text-success'>
    <td class='text text-warning'><div class='testcase'>test_dev_manage_dev_search_by_imei: 测试设备管理-设备搜索-by imei</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link btn btn-xs btn-default" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.1')" >
        error</a>

    <div id='div_ft1.1' class="popup_window">
        <div style='text-align: right;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.1: Traceback (most recent call last):
  File "E:\git\tuqiangOL\testcases\dev_manage\test_case_085_dev_manage_dev_stock.py", line 20, in setUp
    self.driver = AutomateDriver()
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\automate_driver\automate_driver.py", line 15, in __init__
    self.driver = browser('chrome')
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\driver\driver_choose.py", line 7, in browser
    driver = webdriver.Chrome()
  File "C:\Python34\lib\site-packages\selenium\webdriver\chrome\webdriver.py", line 69, in __init__
    desired_capabilities=desired_capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 90, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 177, in start_session
    response = self.execute(Command.NEW_SESSION, capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 236, in execute
    self.error_handler.check_response(response)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 192, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: chrome failed to start
  (Driver info: chromedriver=2.27.440174 (e97a722caafc2d3a8b807ee115bfb307f7d2cfd9),platform=Windows NT 6.1.7600 x86_64)



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='text text-warning'>
    <td>test_case_086_dev_manage_dev_search_by_imei.TestCase086DevManageDevSearchByIMEI</td>
    <td>1</td>
    <td>0</td>
    <td>0</td>
    <td>1</td>
    <td><a class="btn btn-xs btn-primary"href="javascript:showClassDetail('c2',1)">Detail</a></td>
</tr>

<tr id='ft2.1' class='text text-success'>
    <td class='text text-warning'><div class='testcase'>test_dev_manage_dev_search_by_imei: 测试设备管理-设备搜索-by imei</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link btn btn-xs btn-default" onfocus='this.blur();' href="javascript:showTestDetail('div_ft2.1')" >
        error</a>

    <div id='div_ft2.1' class="popup_window">
        <div style='text-align: right;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft2.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft2.1: Traceback (most recent call last):
  File "E:\git\tuqiangOL\testcases\dev_manage\test_case_086_dev_manage_dev_search_by_imei.py", line 20, in setUp
    self.driver = AutomateDriver()
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\automate_driver\automate_driver.py", line 15, in __init__
    self.driver = browser('chrome')
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\driver\driver_choose.py", line 7, in browser
    driver = webdriver.Chrome()
  File "C:\Python34\lib\site-packages\selenium\webdriver\chrome\webdriver.py", line 69, in __init__
    desired_capabilities=desired_capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 90, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 177, in start_session
    response = self.execute(Command.NEW_SESSION, capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 236, in execute
    self.error_handler.check_response(response)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 192, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: chrome failed to start
  (Driver info: chromedriver=2.27.440174 (e97a722caafc2d3a8b807ee115bfb307f7d2cfd9),platform=Windows NT 6.1.7600 x86_64)



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='text text-warning'>
    <td>test_case_087_dev_manage_dev_search_by_dev_name.TestCase087DevManageDevSearchByDevName</td>
    <td>1</td>
    <td>0</td>
    <td>0</td>
    <td>1</td>
    <td><a class="btn btn-xs btn-primary"href="javascript:showClassDetail('c3',1)">Detail</a></td>
</tr>

<tr id='ft3.1' class='text text-success'>
    <td class='text text-warning'><div class='testcase'>test_dev_manage_dev_search_by_dev_name: 测试设备管理-设备搜索-by 设备名称</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link btn btn-xs btn-default" onfocus='this.blur();' href="javascript:showTestDetail('div_ft3.1')" >
        error</a>

    <div id='div_ft3.1' class="popup_window">
        <div style='text-align: right;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft3.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft3.1: Traceback (most recent call last):
  File "E:\git\tuqiangOL\testcases\dev_manage\test_case_087_dev_manage_dev_search_by_dev_name.py", line 20, in setUp
    self.driver = AutomateDriver()
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\automate_driver\automate_driver.py", line 15, in __init__
    self.driver = browser('chrome')
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\driver\driver_choose.py", line 7, in browser
    driver = webdriver.Chrome()
  File "C:\Python34\lib\site-packages\selenium\webdriver\chrome\webdriver.py", line 69, in __init__
    desired_capabilities=desired_capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 90, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 177, in start_session
    response = self.execute(Command.NEW_SESSION, capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 236, in execute
    self.error_handler.check_response(response)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 192, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: chrome failed to start
  (Driver info: chromedriver=2.27.440174 (e97a722caafc2d3a8b807ee115bfb307f7d2cfd9),platform=Windows NT 6.1.7600 x86_64)



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='text text-warning'>
    <td>test_case_088_dev_manage_dev_search_by_dev_type.TestCase088DevManageDevSearchByDevType</td>
    <td>1</td>
    <td>0</td>
    <td>0</td>
    <td>1</td>
    <td><a class="btn btn-xs btn-primary"href="javascript:showClassDetail('c4',1)">Detail</a></td>
</tr>

<tr id='ft4.1' class='text text-success'>
    <td class='text text-warning'><div class='testcase'>test_dev_manage_dev_search_by_dev_type: 测试设备管理-设备搜索-by 设备型号</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link btn btn-xs btn-default" onfocus='this.blur();' href="javascript:showTestDetail('div_ft4.1')" >
        error</a>

    <div id='div_ft4.1' class="popup_window">
        <div style='text-align: right;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft4.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft4.1: Traceback (most recent call last):
  File "E:\git\tuqiangOL\testcases\dev_manage\test_case_088_dev_manage_dev_search_by_dev_type.py", line 20, in setUp
    self.driver = AutomateDriver()
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\automate_driver\automate_driver.py", line 15, in __init__
    self.driver = browser('chrome')
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\driver\driver_choose.py", line 7, in browser
    driver = webdriver.Chrome()
  File "C:\Python34\lib\site-packages\selenium\webdriver\chrome\webdriver.py", line 69, in __init__
    desired_capabilities=desired_capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 90, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 177, in start_session
    response = self.execute(Command.NEW_SESSION, capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 236, in execute
    self.error_handler.check_response(response)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 192, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: chrome failed to start
  (Driver info: chromedriver=2.27.440174 (e97a722caafc2d3a8b807ee115bfb307f7d2cfd9),platform=Windows NT 6.1.7600 x86_64)



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='text text-warning'>
    <td>test_case_089_dev_manage_curr_and_lower_dev_search_by_dev_expired_status.TestCase089DevManageCurrAndLowerDevSearchByDevExpiredStatus</td>
    <td>1</td>
    <td>0</td>
    <td>0</td>
    <td>1</td>
    <td><a class="btn btn-xs btn-primary"href="javascript:showClassDetail('c5',1)">Detail</a></td>
</tr>

<tr id='ft5.1' class='text text-success'>
    <td class='text text-warning'><div class='testcase'>test_dev_manage_curr_and_lower_dev_search_by_dev_expired_status: 测试设备管理-包含下级的设备搜索-by 设备过期状态</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link btn btn-xs btn-default" onfocus='this.blur();' href="javascript:showTestDetail('div_ft5.1')" >
        error</a>

    <div id='div_ft5.1' class="popup_window">
        <div style='text-align: right;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft5.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft5.1: Traceback (most recent call last):
  File "E:\git\tuqiangOL\testcases\dev_manage\test_case_089_dev_manage_curr_and_lower_dev_search_by_dev_expired_status.py", line 19, in setUp
    self.driver = AutomateDriver()
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\automate_driver\automate_driver.py", line 15, in __init__
    self.driver = browser('chrome')
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\driver\driver_choose.py", line 7, in browser
    driver = webdriver.Chrome()
  File "C:\Python34\lib\site-packages\selenium\webdriver\chrome\webdriver.py", line 69, in __init__
    desired_capabilities=desired_capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 90, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 177, in start_session
    response = self.execute(Command.NEW_SESSION, capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 236, in execute
    self.error_handler.check_response(response)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 192, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: chrome failed to start
  (Driver info: chromedriver=2.27.440174 (e97a722caafc2d3a8b807ee115bfb307f7d2cfd9),platform=Windows NT 6.1.7600 x86_64)



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='text text-warning'>
    <td>test_case_090_dev_manage_dev_search_by_vehicle_number.TestCase090DevManageDevSearchByVehicleNumber</td>
    <td>1</td>
    <td>0</td>
    <td>0</td>
    <td>1</td>
    <td><a class="btn btn-xs btn-primary"href="javascript:showClassDetail('c6',1)">Detail</a></td>
</tr>

<tr id='ft6.1' class='text text-success'>
    <td class='text text-warning'><div class='testcase'>test_dev_manage_dev_search_by_vehicle_number: 测试设备管理-设备搜索-by 车牌号</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link btn btn-xs btn-default" onfocus='this.blur();' href="javascript:showTestDetail('div_ft6.1')" >
        error</a>

    <div id='div_ft6.1' class="popup_window">
        <div style='text-align: right;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft6.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft6.1: Traceback (most recent call last):
  File "E:\git\tuqiangOL\testcases\dev_manage\test_case_090_dev_manage_dev_search_by_vehicle_number.py", line 20, in setUp
    self.driver = AutomateDriver()
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\automate_driver\automate_driver.py", line 15, in __init__
    self.driver = browser('chrome')
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\driver\driver_choose.py", line 7, in browser
    driver = webdriver.Chrome()
  File "C:\Python34\lib\site-packages\selenium\webdriver\chrome\webdriver.py", line 69, in __init__
    desired_capabilities=desired_capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 90, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 177, in start_session
    response = self.execute(Command.NEW_SESSION, capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 236, in execute
    self.error_handler.check_response(response)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 192, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: chrome failed to start
  (Driver info: chromedriver=2.27.440174 (e97a722caafc2d3a8b807ee115bfb307f7d2cfd9),platform=Windows NT 6.1.7600 x86_64)



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='text text-warning'>
    <td>test_case_091_dev_manage_curr_and_lower_dev_search_by_car_frame.TestCase091DevManageCurrAndLowerDevSearchByCarFrame</td>
    <td>1</td>
    <td>0</td>
    <td>0</td>
    <td>1</td>
    <td><a class="btn btn-xs btn-primary"href="javascript:showClassDetail('c7',1)">Detail</a></td>
</tr>

<tr id='ft7.1' class='text text-success'>
    <td class='text text-warning'><div class='testcase'>test_dev_manage_curr_and_lower_dev_search_by_car_frame: 测试设备管理-包含下级的设备搜索-by 车架号</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link btn btn-xs btn-default" onfocus='this.blur();' href="javascript:showTestDetail('div_ft7.1')" >
        error</a>

    <div id='div_ft7.1' class="popup_window">
        <div style='text-align: right;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft7.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft7.1: Traceback (most recent call last):
  File "E:\git\tuqiangOL\testcases\dev_manage\test_case_091_dev_manage_curr_and_lower_dev_search_by_car_frame.py", line 20, in setUp
    self.driver = AutomateDriver()
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\automate_driver\automate_driver.py", line 15, in __init__
    self.driver = browser('chrome')
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\driver\driver_choose.py", line 7, in browser
    driver = webdriver.Chrome()
  File "C:\Python34\lib\site-packages\selenium\webdriver\chrome\webdriver.py", line 69, in __init__
    desired_capabilities=desired_capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 90, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 177, in start_session
    response = self.execute(Command.NEW_SESSION, capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 236, in execute
    self.error_handler.check_response(response)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 192, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: chrome failed to start
  (Driver info: chromedriver=2.27.440174 (e97a722caafc2d3a8b807ee115bfb307f7d2cfd9),platform=Windows NT 6.1.7600 x86_64)



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='text text-warning'>
    <td>test_case_092_dev_manage_dev_search_by_SIM.TestCase092DevManageDevSearchBySIM</td>
    <td>1</td>
    <td>0</td>
    <td>0</td>
    <td>1</td>
    <td><a class="btn btn-xs btn-primary"href="javascript:showClassDetail('c8',1)">Detail</a></td>
</tr>

<tr id='ft8.1' class='text text-success'>
    <td class='text text-warning'><div class='testcase'>test_dev_manage_dev_search_by_SIM: 测试设备管理-设备搜索-by SIM</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link btn btn-xs btn-default" onfocus='this.blur();' href="javascript:showTestDetail('div_ft8.1')" >
        error</a>

    <div id='div_ft8.1' class="popup_window">
        <div style='text-align: right;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft8.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft8.1: Traceback (most recent call last):
  File "E:\git\tuqiangOL\testcases\dev_manage\test_case_092_dev_manage_dev_search_by_SIM.py", line 20, in setUp
    self.driver = AutomateDriver()
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\automate_driver\automate_driver.py", line 15, in __init__
    self.driver = browser('chrome')
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\driver\driver_choose.py", line 7, in browser
    driver = webdriver.Chrome()
  File "C:\Python34\lib\site-packages\selenium\webdriver\chrome\webdriver.py", line 69, in __init__
    desired_capabilities=desired_capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 90, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 177, in start_session
    response = self.execute(Command.NEW_SESSION, capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 236, in execute
    self.error_handler.check_response(response)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 192, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: chrome failed to start
  (Driver info: chromedriver=2.27.440174 (e97a722caafc2d3a8b807ee115bfb307f7d2cfd9),platform=Windows NT 6.1.7600 x86_64)



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='text text-warning'>
    <td>test_case_093_dev_manage_dev_search_by_active_status.TestCase093DevManageDevSearchByActiveStatus</td>
    <td>1</td>
    <td>0</td>
    <td>0</td>
    <td>1</td>
    <td><a class="btn btn-xs btn-primary"href="javascript:showClassDetail('c9',1)">Detail</a></td>
</tr>

<tr id='ft9.1' class='text text-success'>
    <td class='text text-warning'><div class='testcase'>test_dev_manage_dev_search_by_active_status: 测试设备管理-设备搜索-by 激活状态</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link btn btn-xs btn-default" onfocus='this.blur();' href="javascript:showTestDetail('div_ft9.1')" >
        error</a>

    <div id='div_ft9.1' class="popup_window">
        <div style='text-align: right;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft9.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft9.1: Traceback (most recent call last):
  File "E:\git\tuqiangOL\testcases\dev_manage\test_case_093_dev_manage_dev_search_by_active_status.py", line 19, in setUp
    self.driver = AutomateDriver()
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\automate_driver\automate_driver.py", line 15, in __init__
    self.driver = browser('chrome')
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\driver\driver_choose.py", line 7, in browser
    driver = webdriver.Chrome()
  File "C:\Python34\lib\site-packages\selenium\webdriver\chrome\webdriver.py", line 69, in __init__
    desired_capabilities=desired_capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 90, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 177, in start_session
    response = self.execute(Command.NEW_SESSION, capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 236, in execute
    self.error_handler.check_response(response)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 192, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: chrome failed to start
  (Driver info: chromedriver=2.27.440174 (e97a722caafc2d3a8b807ee115bfb307f7d2cfd9),platform=Windows NT 6.1.7600 x86_64)



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='text text-warning'>
    <td>test_case_094_dev_manage_dev_search_by_active_time.TestCase094DevManageDevSearchByActiveTime</td>
    <td>1</td>
    <td>0</td>
    <td>0</td>
    <td>1</td>
    <td><a class="btn btn-xs btn-primary"href="javascript:showClassDetail('c10',1)">Detail</a></td>
</tr>

<tr id='ft10.1' class='text text-success'>
    <td class='text text-warning'><div class='testcase'>test_dev_manage_dev_search_by_active_time: 测试设备管理-设备搜索-by 激活时间</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link btn btn-xs btn-default" onfocus='this.blur();' href="javascript:showTestDetail('div_ft10.1')" >
        error</a>

    <div id='div_ft10.1' class="popup_window">
        <div style='text-align: right;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft10.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft10.1: Traceback (most recent call last):
  File "E:\git\tuqiangOL\testcases\dev_manage\test_case_094_dev_manage_dev_search_by_active_time.py", line 20, in setUp
    self.driver = AutomateDriver()
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\automate_driver\automate_driver.py", line 15, in __init__
    self.driver = browser('chrome')
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\driver\driver_choose.py", line 7, in browser
    driver = webdriver.Chrome()
  File "C:\Python34\lib\site-packages\selenium\webdriver\chrome\webdriver.py", line 69, in __init__
    desired_capabilities=desired_capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 90, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 177, in start_session
    response = self.execute(Command.NEW_SESSION, capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 236, in execute
    self.error_handler.check_response(response)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 192, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: chrome failed to start
  (Driver info: chromedriver=2.27.440174 (e97a722caafc2d3a8b807ee115bfb307f7d2cfd9),platform=Windows NT 6.1.7600 x86_64)



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='text text-warning'>
    <td>test_case_095_dev_manage_dev_search_by_plat_expired_time.TestCase095DevManageDevSearchByPlatExpiredTime</td>
    <td>1</td>
    <td>0</td>
    <td>0</td>
    <td>1</td>
    <td><a class="btn btn-xs btn-primary"href="javascript:showClassDetail('c11',1)">Detail</a></td>
</tr>

<tr id='ft11.1' class='text text-success'>
    <td class='text text-warning'><div class='testcase'>test_dev_manage_dev_search_by_plat_expired_time: 测试设备管理-设备搜索-by 平台到期时间</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link btn btn-xs btn-default" onfocus='this.blur();' href="javascript:showTestDetail('div_ft11.1')" >
        error</a>

    <div id='div_ft11.1' class="popup_window">
        <div style='text-align: right;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft11.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft11.1: Traceback (most recent call last):
  File "E:\git\tuqiangOL\testcases\dev_manage\test_case_095_dev_manage_dev_search_by_plat_expired_time.py", line 20, in setUp
    self.driver = AutomateDriver()
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\automate_driver\automate_driver.py", line 15, in __init__
    self.driver = browser('chrome')
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\driver\driver_choose.py", line 7, in browser
    driver = webdriver.Chrome()
  File "C:\Python34\lib\site-packages\selenium\webdriver\chrome\webdriver.py", line 69, in __init__
    desired_capabilities=desired_capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 90, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 177, in start_session
    response = self.execute(Command.NEW_SESSION, capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 236, in execute
    self.error_handler.check_response(response)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 192, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: chrome failed to start
  (Driver info: chromedriver=2.27.440174 (e97a722caafc2d3a8b807ee115bfb307f7d2cfd9),platform=Windows NT 6.1.7600 x86_64)



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='text text-warning'>
    <td>test_case_096_dev_manage_dev_search_by_dev_use_range.TestCase096DevManageDevSearchByDevUseRange</td>
    <td>1</td>
    <td>0</td>
    <td>0</td>
    <td>1</td>
    <td><a class="btn btn-xs btn-primary"href="javascript:showClassDetail('c12',1)">Detail</a></td>
</tr>

<tr id='ft12.1' class='text text-success'>
    <td class='text text-warning'><div class='testcase'>test_dev_manage_dev_search_by_dev_use_range: 测试设备管理-设备搜索-by 设备使用范围</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link btn btn-xs btn-default" onfocus='this.blur();' href="javascript:showTestDetail('div_ft12.1')" >
        error</a>

    <div id='div_ft12.1' class="popup_window">
        <div style='text-align: right;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft12.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft12.1: Traceback (most recent call last):
  File "E:\git\tuqiangOL\testcases\dev_manage\test_case_096_dev_manage_dev_search_by_dev_use_range.py", line 20, in setUp
    self.driver = AutomateDriver()
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\automate_driver\automate_driver.py", line 15, in __init__
    self.driver = browser('chrome')
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\driver\driver_choose.py", line 7, in browser
    driver = webdriver.Chrome()
  File "C:\Python34\lib\site-packages\selenium\webdriver\chrome\webdriver.py", line 69, in __init__
    desired_capabilities=desired_capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 90, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 177, in start_session
    response = self.execute(Command.NEW_SESSION, capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 236, in execute
    self.error_handler.check_response(response)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 192, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: chrome failed to start
  (Driver info: chromedriver=2.27.440174 (e97a722caafc2d3a8b807ee115bfb307f7d2cfd9),platform=Windows NT 6.1.7600 x86_64)



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='text text-warning'>
    <td>test_case_097_dev_manage_curr_and_lower_dev_search_by_active_status.TestCase097DevManageCurrAndLowerDevSearchByActiveStatus</td>
    <td>1</td>
    <td>0</td>
    <td>0</td>
    <td>1</td>
    <td><a class="btn btn-xs btn-primary"href="javascript:showClassDetail('c13',1)">Detail</a></td>
</tr>

<tr id='ft13.1' class='text text-success'>
    <td class='text text-warning'><div class='testcase'>test_dev_manage_curr_and_lower_dev_search_by_active_status: 测试设备管理-包含下级设备的设备搜索-by 设备激活状态</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link btn btn-xs btn-default" onfocus='this.blur();' href="javascript:showTestDetail('div_ft13.1')" >
        error</a>

    <div id='div_ft13.1' class="popup_window">
        <div style='text-align: right;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft13.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft13.1: Traceback (most recent call last):
  File "E:\git\tuqiangOL\testcases\dev_manage\test_case_097_dev_manage_curr_and_lower_dev_search_by_active_status.py", line 20, in setUp
    self.driver = AutomateDriver()
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\automate_driver\automate_driver.py", line 15, in __init__
    self.driver = browser('chrome')
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\driver\driver_choose.py", line 7, in browser
    driver = webdriver.Chrome()
  File "C:\Python34\lib\site-packages\selenium\webdriver\chrome\webdriver.py", line 69, in __init__
    desired_capabilities=desired_capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 90, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 177, in start_session
    response = self.execute(Command.NEW_SESSION, capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 236, in execute
    self.error_handler.check_response(response)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 192, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: chrome failed to start
  (Driver info: chromedriver=2.27.440174 (e97a722caafc2d3a8b807ee115bfb307f7d2cfd9),platform=Windows NT 6.1.7600 x86_64)



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='text text-warning'>
    <td>test_case_098_dev_manage_dev_search_by_dev_type_and_car_frame.TestCase098DevManageDevSearchByDevTypeAndCarFrame</td>
    <td>1</td>
    <td>0</td>
    <td>0</td>
    <td>1</td>
    <td><a class="btn btn-xs btn-primary"href="javascript:showClassDetail('c14',1)">Detail</a></td>
</tr>

<tr id='ft14.1' class='text text-success'>
    <td class='text text-warning'><div class='testcase'>test_dev_manage_dev_search_by_dev_type_and_car_frame: 测试设备管理-包含下级的设备搜索-by 设备类型+车牌号</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link btn btn-xs btn-default" onfocus='this.blur();' href="javascript:showTestDetail('div_ft14.1')" >
        error</a>

    <div id='div_ft14.1' class="popup_window">
        <div style='text-align: right;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft14.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft14.1: Traceback (most recent call last):
  File "E:\git\tuqiangOL\testcases\dev_manage\test_case_098_dev_manage_dev_search_by_dev_type_and_car_frame.py", line 20, in setUp
    self.driver = AutomateDriver()
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\automate_driver\automate_driver.py", line 15, in __init__
    self.driver = browser('chrome')
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\driver\driver_choose.py", line 7, in browser
    driver = webdriver.Chrome()
  File "C:\Python34\lib\site-packages\selenium\webdriver\chrome\webdriver.py", line 69, in __init__
    desired_capabilities=desired_capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 90, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 177, in start_session
    response = self.execute(Command.NEW_SESSION, capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 236, in execute
    self.error_handler.check_response(response)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 192, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: chrome failed to start
  (Driver info: chromedriver=2.27.440174 (e97a722caafc2d3a8b807ee115bfb307f7d2cfd9),platform=Windows NT 6.1.7600 x86_64)



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='text text-warning'>
    <td>test_case_099_dev_manage_dev_search_by_dev_type_and_active_status.TestCase099DevManageDevSearchByDevTypeAndActiveStatus</td>
    <td>1</td>
    <td>0</td>
    <td>0</td>
    <td>1</td>
    <td><a class="btn btn-xs btn-primary"href="javascript:showClassDetail('c15',1)">Detail</a></td>
</tr>

<tr id='ft15.1' class='text text-success'>
    <td class='text text-warning'><div class='testcase'>test_dev_manage_dev_search_by_dev_type_and_expired: 测试设备管理-包含下级的设备搜索-by 设备类型+激活状态（未激活）</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link btn btn-xs btn-default" onfocus='this.blur();' href="javascript:showTestDetail('div_ft15.1')" >
        error</a>

    <div id='div_ft15.1' class="popup_window">
        <div style='text-align: right;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft15.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft15.1: Traceback (most recent call last):
  File "E:\git\tuqiangOL\testcases\dev_manage\test_case_099_dev_manage_dev_search_by_dev_type_and_active_status.py", line 21, in setUp
    self.driver = AutomateDriver()
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\automate_driver\automate_driver.py", line 15, in __init__
    self.driver = browser('chrome')
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\driver\driver_choose.py", line 7, in browser
    driver = webdriver.Chrome()
  File "C:\Python34\lib\site-packages\selenium\webdriver\chrome\webdriver.py", line 69, in __init__
    desired_capabilities=desired_capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 90, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 177, in start_session
    response = self.execute(Command.NEW_SESSION, capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 236, in execute
    self.error_handler.check_response(response)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 192, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: chrome failed to start
  (Driver info: chromedriver=2.27.440174 (e97a722caafc2d3a8b807ee115bfb307f7d2cfd9),platform=Windows NT 6.1.7600 x86_64)



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='text text-warning'>
    <td>test_case_100_dev_manage_dev_search_by_active_time_and_use_range.TestCase100DevManageDevSearchByActiveTimeAndUseRange</td>
    <td>1</td>
    <td>0</td>
    <td>0</td>
    <td>1</td>
    <td><a class="btn btn-xs btn-primary"href="javascript:showClassDetail('c16',1)">Detail</a></td>
</tr>

<tr id='ft16.1' class='text text-success'>
    <td class='text text-warning'><div class='testcase'>test_dev_manage_dev_search_by_active_time_and_use_range: 测试设备管理-包含下级的设备搜索-by 激活状态（已激活）+激活时间段+使用范围</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link btn btn-xs btn-default" onfocus='this.blur();' href="javascript:showTestDetail('div_ft16.1')" >
        error</a>

    <div id='div_ft16.1' class="popup_window">
        <div style='text-align: right;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft16.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft16.1: Traceback (most recent call last):
  File "E:\git\tuqiangOL\testcases\dev_manage\test_case_100_dev_manage_dev_search_by_active_time_and_use_range.py", line 21, in setUp
    self.driver = AutomateDriver()
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\automate_driver\automate_driver.py", line 15, in __init__
    self.driver = browser('chrome')
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\driver\driver_choose.py", line 7, in browser
    driver = webdriver.Chrome()
  File "C:\Python34\lib\site-packages\selenium\webdriver\chrome\webdriver.py", line 69, in __init__
    desired_capabilities=desired_capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 90, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 177, in start_session
    response = self.execute(Command.NEW_SESSION, capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 236, in execute
    self.error_handler.check_response(response)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 192, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: chrome failed to start
  (Driver info: chromedriver=2.27.440174 (e97a722caafc2d3a8b807ee115bfb307f7d2cfd9),platform=Windows NT 6.1.7600 x86_64)



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='text text-warning'>
    <td>test_case_101_dev_manage_dev_complex_search.TestCase101DevManageDevComplexSearch</td>
    <td>1</td>
    <td>0</td>
    <td>0</td>
    <td>1</td>
    <td><a class="btn btn-xs btn-primary"href="javascript:showClassDetail('c17',1)">Detail</a></td>
</tr>

<tr id='ft17.1' class='text text-success'>
    <td class='text text-warning'><div class='testcase'>test_dev_manage_dev_complex_search: 测试设备管理-设备搜索-by imei+设备名称+设备型号+车牌号+车架号+SIM</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link btn btn-xs btn-default" onfocus='this.blur();' href="javascript:showTestDetail('div_ft17.1')" >
        error</a>

    <div id='div_ft17.1' class="popup_window">
        <div style='text-align: right;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft17.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft17.1: Traceback (most recent call last):
  File "E:\git\tuqiangOL\testcases\dev_manage\test_case_101_dev_manage_dev_complex_search.py", line 20, in setUp
    self.driver = AutomateDriver()
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\automate_driver\automate_driver.py", line 15, in __init__
    self.driver = browser('chrome')
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\driver\driver_choose.py", line 7, in browser
    driver = webdriver.Chrome()
  File "C:\Python34\lib\site-packages\selenium\webdriver\chrome\webdriver.py", line 69, in __init__
    desired_capabilities=desired_capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 90, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 177, in start_session
    response = self.execute(Command.NEW_SESSION, capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 236, in execute
    self.error_handler.check_response(response)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 192, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: chrome failed to start
  (Driver info: chromedriver=2.27.440174 (e97a722caafc2d3a8b807ee115bfb307f7d2cfd9),platform=Windows NT 6.1.7600 x86_64)



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='text text-warning'>
    <td>test_case_102_dev_manage_dev_operate.TestCase102DevManageDevOperate</td>
    <td>1</td>
    <td>0</td>
    <td>0</td>
    <td>1</td>
    <td><a class="btn btn-xs btn-primary"href="javascript:showClassDetail('c18',1)">Detail</a></td>
</tr>

<tr id='ft18.1' class='text text-success'>
    <td class='text text-warning'><div class='testcase'>test_dev_manage_dev_operate: 测试设备管理-单个设备操作-编辑、查看位置、查看告警</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link btn btn-xs btn-default" onfocus='this.blur();' href="javascript:showTestDetail('div_ft18.1')" >
        error</a>

    <div id='div_ft18.1' class="popup_window">
        <div style='text-align: right;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft18.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft18.1: Traceback (most recent call last):
  File "E:\git\tuqiangOL\testcases\dev_manage\test_case_102_dev_manage_dev_operate.py", line 20, in setUp
    self.driver = AutomateDriver()
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\automate_driver\automate_driver.py", line 15, in __init__
    self.driver = browser('chrome')
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\driver\driver_choose.py", line 7, in browser
    driver = webdriver.Chrome()
  File "C:\Python34\lib\site-packages\selenium\webdriver\chrome\webdriver.py", line 69, in __init__
    desired_capabilities=desired_capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 90, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 177, in start_session
    response = self.execute(Command.NEW_SESSION, capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 236, in execute
    self.error_handler.check_response(response)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 192, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: chrome failed to start
  (Driver info: chromedriver=2.27.440174 (e97a722caafc2d3a8b807ee115bfb307f7d2cfd9),platform=Windows NT 6.1.7600 x86_64)



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='text text-warning'>
    <td>test_case_103_dev_manage_dev_batch_send_instr_single.TestCase103DevManageDevBatchSendInstrSingle</td>
    <td>1</td>
    <td>0</td>
    <td>0</td>
    <td>1</td>
    <td><a class="btn btn-xs btn-primary"href="javascript:showClassDetail('c19',1)">Detail</a></td>
</tr>

<tr id='ft19.1' class='text text-success'>
    <td class='text text-warning'><div class='testcase'>test_dev_manage_dev_batch_send_instr_single: 测试设备管理-设备批量操作-选中单条设备发送指令</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link btn btn-xs btn-default" onfocus='this.blur();' href="javascript:showTestDetail('div_ft19.1')" >
        error</a>

    <div id='div_ft19.1' class="popup_window">
        <div style='text-align: right;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft19.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft19.1: Traceback (most recent call last):
  File "E:\git\tuqiangOL\testcases\dev_manage\test_case_103_dev_manage_dev_batch_send_instr_single.py", line 20, in setUp
    self.driver = AutomateDriver()
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\automate_driver\automate_driver.py", line 15, in __init__
    self.driver = browser('chrome')
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\driver\driver_choose.py", line 7, in browser
    driver = webdriver.Chrome()
  File "C:\Python34\lib\site-packages\selenium\webdriver\chrome\webdriver.py", line 69, in __init__
    desired_capabilities=desired_capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 90, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 177, in start_session
    response = self.execute(Command.NEW_SESSION, capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 236, in execute
    self.error_handler.check_response(response)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 192, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: chrome failed to start
  (Driver info: chromedriver=2.27.440174 (e97a722caafc2d3a8b807ee115bfb307f7d2cfd9),platform=Windows NT 6.1.7600 x86_64)



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='text text-warning'>
    <td>test_case_104_dev_manage_dev_batch_send_instr_many.TestCase104DevManageDevBatchSendInstrMany</td>
    <td>1</td>
    <td>0</td>
    <td>0</td>
    <td>1</td>
    <td><a class="btn btn-xs btn-primary"href="javascript:showClassDetail('c20',1)">Detail</a></td>
</tr>

<tr id='ft20.1' class='text text-success'>
    <td class='text text-warning'><div class='testcase'>test_dev_manage_dev_batch_send_instr_many: 测试设备管理-设备批量操作-选中多条设备发送指令</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link btn btn-xs btn-default" onfocus='this.blur();' href="javascript:showTestDetail('div_ft20.1')" >
        error</a>

    <div id='div_ft20.1' class="popup_window">
        <div style='text-align: right;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft20.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft20.1: Traceback (most recent call last):
  File "E:\git\tuqiangOL\testcases\dev_manage\test_case_104_dev_manage_dev_batch_send_instr_many.py", line 20, in setUp
    self.driver = AutomateDriver()
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\automate_driver\automate_driver.py", line 15, in __init__
    self.driver = browser('chrome')
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\driver\driver_choose.py", line 7, in browser
    driver = webdriver.Chrome()
  File "C:\Python34\lib\site-packages\selenium\webdriver\chrome\webdriver.py", line 69, in __init__
    desired_capabilities=desired_capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 90, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 177, in start_session
    response = self.execute(Command.NEW_SESSION, capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 236, in execute
    self.error_handler.check_response(response)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 192, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: chrome failed to start
  (Driver info: chromedriver=2.27.440174 (e97a722caafc2d3a8b807ee115bfb307f7d2cfd9),platform=Windows NT 6.1.7600 x86_64)



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='text text-warning'>
    <td>test_case_105_dev_manage_dev_batch_set_mode_single.TestCase105DevManageDevBatchSetModeSingle</td>
    <td>1</td>
    <td>0</td>
    <td>0</td>
    <td>1</td>
    <td><a class="btn btn-xs btn-primary"href="javascript:showClassDetail('c21',1)">Detail</a></td>
</tr>

<tr id='ft21.1' class='text text-success'>
    <td class='text text-warning'><div class='testcase'>test_dev_manage_dev_batch_set_mode_single: 测试设备管理-设备批量操作-选中单条设备设置工作模式</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link btn btn-xs btn-default" onfocus='this.blur();' href="javascript:showTestDetail('div_ft21.1')" >
        error</a>

    <div id='div_ft21.1' class="popup_window">
        <div style='text-align: right;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft21.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft21.1: Traceback (most recent call last):
  File "E:\git\tuqiangOL\testcases\dev_manage\test_case_105_dev_manage_dev_batch_set_mode_single.py", line 20, in setUp
    self.driver = AutomateDriver()
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\automate_driver\automate_driver.py", line 15, in __init__
    self.driver = browser('chrome')
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\driver\driver_choose.py", line 7, in browser
    driver = webdriver.Chrome()
  File "C:\Python34\lib\site-packages\selenium\webdriver\chrome\webdriver.py", line 69, in __init__
    desired_capabilities=desired_capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 90, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 177, in start_session
    response = self.execute(Command.NEW_SESSION, capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 236, in execute
    self.error_handler.check_response(response)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 192, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: chrome failed to start
  (Driver info: chromedriver=2.27.440174 (e97a722caafc2d3a8b807ee115bfb307f7d2cfd9),platform=Windows NT 6.1.7600 x86_64)



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='text text-warning'>
    <td>test_case_106_dev_manage_dev_batch_set_mode_many.TestCase106DevManageDevBatchSetModeMany</td>
    <td>1</td>
    <td>0</td>
    <td>0</td>
    <td>1</td>
    <td><a class="btn btn-xs btn-primary"href="javascript:showClassDetail('c22',1)">Detail</a></td>
</tr>

<tr id='ft22.1' class='text text-success'>
    <td class='text text-warning'><div class='testcase'>test_dev_manage_dev_batch_set_mode_single: 测试设备管理-设备批量操作-全选设备设置工作模式</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link btn btn-xs btn-default" onfocus='this.blur();' href="javascript:showTestDetail('div_ft22.1')" >
        error</a>

    <div id='div_ft22.1' class="popup_window">
        <div style='text-align: right;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft22.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft22.1: Traceback (most recent call last):
  File "E:\git\tuqiangOL\testcases\dev_manage\test_case_106_dev_manage_dev_batch_set_mode_many.py", line 20, in setUp
    self.driver = AutomateDriver()
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\automate_driver\automate_driver.py", line 15, in __init__
    self.driver = browser('chrome')
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\driver\driver_choose.py", line 7, in browser
    driver = webdriver.Chrome()
  File "C:\Python34\lib\site-packages\selenium\webdriver\chrome\webdriver.py", line 69, in __init__
    desired_capabilities=desired_capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 90, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 177, in start_session
    response = self.execute(Command.NEW_SESSION, capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 236, in execute
    self.error_handler.check_response(response)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 192, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: chrome failed to start
  (Driver info: chromedriver=2.27.440174 (e97a722caafc2d3a8b807ee115bfb307f7d2cfd9),platform=Windows NT 6.1.7600 x86_64)



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='text text-warning'>
    <td>test_case_107_dev_manage_dev_batch_set_mode_add_mode.TestCase107DevManageDevBatchSetModeAddMode</td>
    <td>1</td>
    <td>0</td>
    <td>0</td>
    <td>1</td>
    <td><a class="btn btn-xs btn-primary"href="javascript:showClassDetail('c23',1)">Detail</a></td>
</tr>

<tr id='ft23.1' class='text text-success'>
    <td class='text text-warning'><div class='testcase'>test_dev_manage_dev_batch_set_mode_add_mode: 测试设备管理-设备批量操作-设置工作模式-新建工作模板</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link btn btn-xs btn-default" onfocus='this.blur();' href="javascript:showTestDetail('div_ft23.1')" >
        error</a>

    <div id='div_ft23.1' class="popup_window">
        <div style='text-align: right;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft23.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft23.1: Traceback (most recent call last):
  File "E:\git\tuqiangOL\testcases\dev_manage\test_case_107_dev_manage_dev_batch_set_mode_add_mode.py", line 20, in setUp
    self.driver = AutomateDriver()
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\automate_driver\automate_driver.py", line 15, in __init__
    self.driver = browser('chrome')
  File "C:\Windows\system32\config\systemprofile\.jenkins\workspace\tuqiangol\driver\driver_choose.py", line 7, in browser
    driver = webdriver.Chrome()
  File "C:\Python34\lib\site-packages\selenium\webdriver\chrome\webdriver.py", line 69, in __init__
    desired_capabilities=desired_capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 90, in __init__
    self.start_session(desired_capabilities, browser_profile)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 177, in start_session
    response = self.execute(Command.NEW_SESSION, capabilities)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 236, in execute
    self.error_handler.check_response(response)
  File "C:\Python34\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 192, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: chrome failed to start
  (Driver info: chromedriver=2.27.440174 (e97a722caafc2d3a8b807ee115bfb307f7d2cfd9),platform=Windows NT 6.1.7600 x86_64)



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

    </tbody>
    <tfoot>
        <tr id='total_row'>
            <td>Total</td>
            <td>23</td>
            <td class="text text-success">0</td>
            <td class="text text-danger">0</td>
            <td class="text text-warning">23</td>
            <td>&nbsp;</td>
        </tr>
    </tfoot>
</table>

    <div id='ending'>&nbsp;</div>
</div>

</body>
</html>
